### ISSUE TITLE
Speaker-Diarization for 2 person conversation

### ISSUE THREAD

**Issue by ArvindSharma18** [taylorlu/Speaker-Diarization#55]
@taylorlu, I would like to appreciate your effort for this repo! I have a small doubt though while trying the Speaker Diarization for .wav file with 2 speakers, I am getting output for 4 different speakers. I would really like to know if there is any way we can change the number of speakers without starting from scratch? It would be really helpful for me!

**Comment by taylorlu** [comment]
No, the model does not support specifying the speaker number, the result all depends on the trained model (speaker and uisrnn) you provided.

**Comment by ArvindSharma18** [comment]
Thanks for your quick response.

**Comment by ArvindSharma18** [comment]
Hi, I tried to generate my own embedding using generate_embeddings.py with tensorflow 1.x , pytorch 1.3 and keras 2.2.4, but I am unable to run it because of compatibility issues. Can you help with suitable versions to run the code?
